bot/resources/evergreen/py_topics.yaml

```
@@ -23,6 +23,18 @@
     - When you were first learning, what is a resource you wish you had?
     - What is something you know now, that you wish you knew when starting out?
     - What is something simple that you still error on today?
+    - What do you plan on eventually achieving with python?
+    - Is python your first programming language? If not, what is it?
+    - What's your favourite aspect of python development? (Backend, web dev, game dev, machine learning, ai, etc.)
+    - In what ways has python discord helped you with python?
+    - Are you currently using Python professionally, for education, or as a hobby?
+    - What is your process when you decide to start a project in python?
+    - Have you ever been unable to finish a python project? What is it and why?
+    - How often do you program in python?
+    - How would you learn a new library if needed to do so?
```

```suggestion
    - How would you approach learning a new Python library?
```
Maybe, previous was fine though as well